### PR TITLE
Locale independent float2string

### PR DESCRIPTION
**Pull request by hjanetzek**
https://github.com/tangrams/tangram-es/issues/1396

### CODE REVIEW

**Comment by hjanetzek** [comment]
@matteblair could you give this a look? Not yet complete but allows to load bubble-wrap independent of LC_NUMERIC. Would be good if we could get something like this into the tizen update next week

**Comment by matteblair** [comment]
As I understand it, the bugs we saw were caused by the behavior of the `to_string` functions, not the `stof`/`stod` functions. The fixes to `to_string` don't depend on the V8 conversion functions, right? What is the value of changing `stod` to use these new functions?

**Comment by hjanetzek** [comment]
@matteblair unfortunately it's both. just to_string would be too simple

**Comment by matteblair** [comment]
Oh wait, is the `stod` change necessary for parsing numbers correctly from the scene file? That is, for correctly interpreting `1.000` and `1,000`?

**Comment by hjanetzek** [comment]
yeah..

**Comment by matteblair** [comment]
Now that I've had time to get my head around this problem, I think it makes sense to move ahead with this. I think we can make a few improvements before merging:
 - Choose a more descriptive name for `fixfloat.h`, maybe `stringConversion.h`
 - Use the V8 conversion function for the `to_string` functions (They seem easy to use, and I would guess they're more efficient than iostreams).

File: platforms/linux/src/main.cpp [hunk 1]
```diff
@@ -22,6 +22,10 @@ int main(int argc, char* argv[]) {
             break;
         }
     }
+    char *loc = setlocale(LC_ALL, "de_DE.utf-8");
+    logMsg("LOCALE %s\n", loc);
+    loc = setlocale(LC_ALL, NULL);
+    logMsg("LOCALE %s - %f\n", loc, 435145.342134);
```
Comment (matteblair) [inline platforms/linux/src/main.cpp]: Ah we should also remove this before we forget about it and confuse ourselves later :)

**Pull request merged**